tests/python/unittest/test_ndarray.py

```diff
@@ -1687,8 +1687,10 @@ def test_zero_from_numpy():
             mx.test_utils.assert_almost_equal(np_array, mx_array.asnumpy())
     np_array = arrays[0]
     mx_array = mx.nd.from_numpy(np_array)
-    np_array[2, 1] = 0
-    mx.test_utils.assert_almost_equal(np_array, mx_array.asnumpy())
+    try:
+        np_array[2, 1] = 0
+    except ValueError:
+        pass
```

Comment:
Are you looking for assert_raises?

Reply:
Thanks for the point, i was not aware of this API before :-) In this testcase, this could be somewhat overkill for this piece of code.

Reply:
assert_raises should be appropriate in that the test case should not pass if the error is not thrown.

Reply:
@szha Will do!